pybop/_classification.py

```diff
@@ -0,0 +1,136 @@
+from typing import Optional
+
+import numpy as np
+
+from pybop import OptimisationResult
+
+
+def classify_using_Hessian(
```

Comment:
A bit pedantic, but in line with Pep8 standards:
```suggestion
def classify_using_hessian(
```